src/clients/dataApiClient.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { describe, it } from "mocha";
+import { DataApiClient, MapiClient } from "../clients";
+import { MockHttpClient } from "./../../tests/mocks";
+import { StaticAuthenticator } from "./../components/staticAuthenticator";
+import { StaticSettingsProvider } from "./../components/staticSettingsProvider";
+import { assert } from "chai";
+import { Utils } from "../utils";
+import * as Constants from "./../constants";
+import { AccessToken } from "../authentication";
+
+interface Validity {
+    isValid: boolean;
+}
+
+//TODO: https://msazure.visualstudio.com/One/_workitems/edit/15015393
+describe("Data API Client", async () => {
```

Comment:
this is the small test to verify that client goes to correct url on retriving and receives the response properly. I would leave it as it is as it may help if the client will be changed to use any other url test will fail.